oak/common/app_config.cc

```diff
@@ -34,23 +34,21 @@ constexpr int16_t kDefaultGrpcPort = 8080;
 
 // Conventional names for the configuration of Nodes.
 constexpr char kAppConfigName[] = "app";
-constexpr char kAppEntrypointName[] = "oak_main";
-constexpr char kLogConfigName[] = "log";
-constexpr char kStorageConfigName[] = "storage";
-constexpr char kGrpcClientConfigName[] = "grpc-client";
+constexpr char kAppWasmModuleName[] = "oak_module";
+constexpr char kAppWasmEntrypointName[] = "oak_main";
 
 }  // namespace
 
 std::unique_ptr<ApplicationConfiguration> DefaultConfig(const std::string& module_bytes) {
   auto config = absl::make_unique<ApplicationConfiguration>();
   config->set_grpc_port(kDefaultGrpcPort);
+  (*config->mutable_wasm_modules())[kAppWasmModuleName] = module_bytes;
```

Comment:
No, my understanding is that this will create the entry if it's not there (it seems to work so I guess that's fine?)

Let me know if I'm doing it wrong though